snap7/types.py

```diff
@@ -212,6 +237,16 @@ class S7DataItem(ctypes.Structure):
         ('pData', ctypes.POINTER(ctypes.c_uint8))
     ]
 
+    def __str__(self) -> str:
```

Comment:
i dont think this should be a multiline representation. This will give problems if you use this string representation in logging for example.

Reply:
Ok, i will switch to the <__class__ (attr=val)> repr